sources/operator.1.12.0.pxc-operator.json

```diff
@@ -0,0 +1,198 @@
+            "critical": false
+          },
+          "5.7.31-31.45": {
+            "image_path": "percona/percona-xtradb-cluster:5.7.31-31.45",
+            "image_hash": "3852cef43cc0c6aa791463ba6279e59dcdac3a4fb1a5616c745c1b3c68041dc2",
+            "status": "available",
+            "critical": false
+          },
+          "5.7.30-31.43": {
+            "image_path": "percona/percona-xtradb-cluster:5.7.30-31.43",
+            "image_hash": "b03a060e9261b37288a2153c78f86dcfc53367c36e1bcdcae046dd2d0b0721af",
+            "status": "available",
+            "critical": false
+          },
+          "5.7.29-31.43": {
+            "image_path": "percona/percona-xtradb-cluster:5.7.29-31.43",
+            "image_hash": "85fb479de073770280ae601cf3ec22dc5c8cca4c8b0dc893b09503767338e6f9",
+            "status": "available",
+            "critical": false
+          },
+          "5.7.28-31.41.2": {
+            "image_path": "percona/percona-xtradb-cluster:5.7.28-31.41.2",
+            "image_hash": "fccd6525aaeedb5e436e9534e2a63aebcf743c043526dd05dba8519ebddc8b30",
+            "status": "available",
+            "critical": true
+          },
+          "5.7.27-31.39": {
+            "image_path": "percona/percona-xtradb-cluster:5.7.27-31.39",
+            "image_hash": "7d8eb4d2031c32c6e96451655f359d8e5e8e047dc95bada9a28c41c158876c26",
+            "status": "available",
+            "critical": false
+          },
+          "5.7.26-31.37": {
+            "image_path": "percona/percona-xtradb-cluster:5.7.26-31.37",
+            "image_hash": "9d43d8e435e4aca5c694f726cc736667cb938158635c5f01a0e9412905f1327f",
+            "status": "available",
+            "critical": false
+          }
+        },
+        "pmm": {
+          "2.32.0": {
+            "image_path": "percona/pmm-client:2.32.0",
+            "image_hash": "ee2f3db541857e0a71633270596933441c4be579ce8e33c22cf150ead4f3622f",
+            "status": "recommended",
+            "critical": false
+          }
+        },
+        "proxysql": {
+          "2.4.3-1.1": {
+            "image_path": "percona/percona-xtradb-cluster-operator:1.12.0-proxysql",
+            "image_hash": "c40ed032f87976a9bf3919bfbfc842e3626fcb5532d7994d9bb91139da359ea3",
+            "status": "recommended",
+            "critical": false
+          }
+        },
+        "haproxy": {
+          "2.5.6": {
+            "image_path": "percona/percona-xtradb-cluster-operator:1.12.0-haproxy",
+            "image_hash": "18ade9b226d651a601b66ef9ded5720d73ee0c9ebf1a31f323d2e71498ee15bb",
+            "status": "recommended",
+            "critical": false
+          }
+        },
+        "backup": {
+          "8.0.29": {
+            "image_path": "percona/percona-xtradb-cluster-operator:1.12.0-pxc8.0-backup",
+            "image_hash": "b737eb09ae878b82d7f654925b466868bf7e914a7f197c3fe3e278cdbc625a1f",
+            "status": "recommended",
+            "critical": false
+          },
+          "2.4.26-1": {
```

Comment:
Seems 2.4.26 is official version: https://docs.percona.com/percona-xtrabackup/2.4/release-notes/2.4/2.4.26.html